app/assets/javascripts/pages/installations.coffee

```diff
@@ -56,9 +46,9 @@ generateApiKey = (data) ->
 
 handleEditDetails = (website_id) ->
   $.getJSON Routes.api_v1_website_path(website_id), (data) ->
-    $('#owner').render data, directive
+    $('#owner').render data
```

Comment:
aia o foloseam pentru remove website_member, dar spuneai ca nu trebuie sa facem asta in edit website